### PR TITLE
Updating lstcontainers and r0_to_dl1 to add Hillas uncertainties fields

### PR DESCRIPTION
**Pull request by andres-baquero**
Adding width and length uncertainties already calculated by ctapipe to lstcontainers and r0_to_dl0 script

### CODE REVIEW

**Review by maxnoe — DISMISSED** [review]

**Comment by maxnoe** [comment]
Is this also needed in dl1ab?

**Comment by andres-baquero** [comment]
> Is this also needed in dl1ab?

You mean to update these fields in scripts/lstchain_dl1ab.py ?

**Comment by andres-baquero** [comment]
> Yes, exactly.

Yes, I think that I must update there too, I can add the fields on:

parameters_to_update and in the moment that the file is filled.

Can I proceed with the changes ? 

and the

**Comment by maxnoe** [comment]
@andres-baquero You have an undefined variable "foclen"

File: lstchain/scripts/lstchain_dl1ab.py
Comment (maxnoe) [inline lstchain/scripts/lstchain_dl1ab.py:253]: replace foclen -> focal_length (see line above)

File: lstchain/scripts/lstchain_dl1ab.py
Comment (maxnoe) [inline lstchain/scripts/lstchain_dl1ab.py:255]: same as above

Comment (andres-baquero) [reply lstchain/scripts/lstchain_dl1ab.py]: I've made the suggested changes

**Review by maxnoe — APPROVED** [review]

**Comment by moralejo** [comment]
> Hi, why this PR was not merged yet on the v.0.7.4 ?

No particular reason, I am afraid it was just forgotten. Can you merge PRs yourself after approval? (I don't know if you need some  privileges).

**Comment by andres-baquero** [comment]
> > Hi, why this PR was not merged yet on the v.0.7.4 ?
> 
> No particular reason, I am afraid it was just forgotten. Can you merge PRs yourself after approval? (I don't know if you need some privileges).

No I can't merge the PR. I need more privileges.

**Comment by moralejo** [comment]
> > No particular reason, I am afraid it was just forgotten. Can you merge PRs yourself after approval? (I don't know if you need some privileges).
> 
> No I can't merge the PR. I need more privileges.

Ok, in that case, I'd suggest everyone keeps en eye on his/her PRs to make sure they're merged as soon as they are approved (please ask @rlopezcoto and myself). I will merge this one now.

**Pull request merged**
